app/views/player_games/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="hero-unit">
   <%= player_avatar(@player) %>
-  <h1><%= link_to @player.name, player_path(@player) %> <strong><%= @player.ratings.find_by_game_id(@game).value %></strong></h1>
+  <h1><%= link_to @player.name, player_path(@player) %> <strong><%= @player.ratings.find_by_game_id(@game)&.value %></strong></h1>
```

Comment:
A bug fix - when a player has no game recorded, `find_by_game_id` returns `nil`, and that leads to an error page.